### PR TITLE
Replace logging screenshots with code-blocks

### PR DESCRIPTION
**Pull request by EdwinVW**
Fix for issue #15.

### CODE REVIEW

**Comment by EdwinVW** [comment]
@mthmulders This PR changes all logging screenshots to text-based code-blocks to fix issue #15.
For all the images that were shared between the .NET and Java instructions, I have already updated them in both versions!

File: Assignment01/step-by-step.md [hunk 1]
```diff
@@ -81,7 +97,27 @@ Now you can test whether you can call the VehicleRegistrationService. You can do
...
+   info: System.Net.Http.HttpClient.Default.LogicalHandler[100]
+         Start processing HTTP request GET http://localhost:6002/vehicleinfo/RT-318-K
+   info: System.Net.Http.HttpClient.Default.ClientHandler[100]
+         Sending HTTP request GET http://localhost:6002/vehicleinfo/RT-318-K
+   info: System.Net.Http.HttpClient.Default.ClientHandler[101]
+         Received HTTP response headers after 148.5761ms - 200
+   info: System.Net.Http.HttpClient.Default.LogicalHandler[101]
+         End processing HTTP request after 156.5675ms - 200
+   info: FineCollectionService.Controllers.CollectionController[0]
+         Sent speeding ticket to Cassi Dakes. Road: A12, Licensenumber: RT-318-K, Vehicle: Mercedes SLK, Violation: 15 Km/h, Fine: 130 Euro, On: 20-09-2020 at 08:33:41.
```
Comment (mthmulders) [inline Assignment01/step-by-step.md:119]: Nice! This line was cropped out of the screenshot, now it's here in its entirety.

File: Assignment01/step-by-step.md [hunk 2]
```diff
@@ -35,7 +35,11 @@ Now you can test whether you can call the VehicleRegistrationService. You can do
 
 1. Click on `Send request` in the file to send a request to the API:
 
-   ![REST client](img/rest-client.png)
+   ```http
```
Comment (mthmulders) [inline Assignment01/step-by-step.md:38]: I'm not sure this is helpful. The screenshot was very clear for those who could read it, as it pointed out where exactly that button is located. The textual representation of the screenshot lost that info. I would say a screenshot makes more sense here than a textual representation of it.

Comment (mthmulders) [reply Assignment01/step-by-step.md:38]: (also applies to the Java guide, by the way)

**Review by mthmulders — COMMENTED** [review]
I'm happy with these changes, except for one spot ;-)

File: Assignment01/step-by-step.md (hunk 2 above)
Comment (EdwinVW) [reply Assignment01/step-by-step.md:38]: Did you not notice the awesome sauce ASCII art arrow that indicates where the `Send request` button is????

No, I agree. I will put the screenshot back.

**Pull request merged**